TypeCobol.Test/SemanticDomain/BadRenames0.cbl

```diff
@@ -0,0 +1,30 @@
+﻿       IDENTIFICATION DIVISION.
```

Comment:
Yes but as I said when a use a specific test is that I cannot do that with existing test.

Reply:
Enhancing existing tests I have to perfomr double checks ou double work, for TypeCobol and and DFA Framework, this is a lot of work.

Reply:
I yes but there are specific tests for DFA, I what I can do is to create needed files in TypeCobol.Test\Parser\Programs without creating a test for TypeCobol. Because enhancing existing tests I have to perform double checks and double work, for TypeCobol and and DFA Framework, this is a lot of work.